cmd/gc_test.go

```diff
@@ -43,10 +71,21 @@ func TestGc(t *testing.T) {
 		}
 	}
 
-	fsckArgs := []string{"", "gc", metaUrl}
-	err := Main(fsckArgs)
+	strEnvSkippedTime := os.Getenv("JFS_GC_SKIPPEDTIME")
+	t.Logf("JFS_GC_SKIPPEDTIME is %s\n",strEnvSkippedTime)
```

Comment:
Logf will add new line in the end, no `\n` needed here.

Reply:
ok